Kudu.Services/Util/DotNetHelper.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Kudu.Core.Helpers;
+
+namespace Kudu.Services.Performance
+{
+    /// <summary>
+    /// Helper class to check if this is a .NET Core blessed image
+    /// </summary>
+    public class DotNetHelper
+    {
+        /// <summary>
+        /// Returns TRUE if the container is running a .NET Core blessed image and
+        /// if the WEBSITE_USE_DOTNET_MONITOR is set to TRUE
+        /// </summary>
+        /// <returns></returns>
+        public static bool IsDotNetMonitorEnabled()
+        {
+            if (OSDetector.IsOnWindows())
+            {
+                return true;
+            }
+
+            string val = Environment.GetEnvironmentVariable("WEBSITE_USE_DOTNET_MONITOR");
+            if (!string.IsNullOrWhiteSpace(val))
+            {
+                string stack = Environment.GetEnvironmentVariable("WEBSITE_STACK");
```

Comment:
We should use `FRAMEWORK` here. Is the app setting `WEBSITE_USE_DOTNET_MONITOR` same as what we use to enable .NET Monitor in LWAS?

Reply:
Switched to using `FRAMEWORK`. Yes `WEBSITE_USE_DOTNET_MONITOR` is the app setting we use to enable .NET Monitor in LWAS